### DISCUSSION TITLE
mix tailwind.install unable to fetch tailwindcss from releases

### DISCUSSION THREAD

**Discussion by drobban** [tailwindlabs/tailwindcss#10589] [Help] [answered]
**What version of Tailwind CSS are you using?**

v3.2.6

**What build tool (or framework if it abstracts the build tool) are you using?**

Elixir - Phoenix - tailwind

**What version of Phoenix - Elixir  IEx are you using?**
Phoenix v1.6.5
IEx 1.14.2 (compiled with Erlang/OTP 24)

**What browser are you using?**

Chrome

**What operating system are you using?**

FreeBSD 13.1

**Reproduction URL**
https://github.com/tailwindlabs/tailwindcss/releases/download/v3.2.6/tailwindcss-freebsd-x64
**Describe your issue**
Unable to install tailwind with `mix tailwind.install`

(RuntimeError) couldn't fetch https://github.com/tailwindlabs/tailwindcss/releases/download/v3.2.6/tailwindcss-freebsd-x64

If there is somehow a way I can help in resolving the issue - just give me some directions and I'll start digging.

=)

**Comment by thecrypticace** [comment]
We do not build releases for FreeBSD. We use a tool called [pkg](https://github.com/vercel/pkg) that allows us to build standalone node binaries but FreeBSD while _possible_ requires compiling node from source for FreeBSD. NodeJS themselves do not offer binaries for FreeBSD and instead defer to FreeBSDs own package registry.

However, I _believe_ that FreeBSD offers some level of Linux binary compatibility. In which case the [tailwindcss-linux-x64](https://github.com/tailwindlabs/tailwindcss/releases/download/v3.2.6/tailwindcss-linux-x64) _might_ be usable so you would need to download that, possibly rename it, and place it in the appropriate folder for Phoenix to pick it up.

**Reply by drobban** [reply]
Thank you for the clarification =)

**Comment by drobban** [comment, marked as answer]
for future people trying to do the same and stumbles over this.

Here is a better answer
https://github.com/phoenixframework/tailwind/pull/71

dch has put up a build for freebsd that runs and describes how to install it with phoenix-tailwindcss in the pr-discussion

**Comment by SeanShubin** [comment]
I am running into a similar problem on maxOS.  It is complaining about an unknown certificate authority.  I reported it with phoenix at https://github.com/phoenixframework/phoenix/issues/5339 , but if it is just a matter of the certificate authority not being set up correctly with tailwind, perhaps this can be fixed on the tailwind side.  Here is the output I get:

```
mix tailwind.install

11:22:51.793 [debug] Downloading tailwind from https://github.com/tailwindlabs/tailwindcss/releases/download/v3.2.4/tailwindcss-macos-x64

11:22:52.238 [notice] TLS :client: In state :wait_cert_cr at ssl_handshake.erl:2111 generated CLIENT ALERT: Fatal - Unknown CA

** (RuntimeError) couldn't fetch https://github.com/tailwindlabs/tailwindcss/releases/download/v3.2.4/tailwindcss-macos-x64: {:error, {:failed_connect, [{:to_address, {'github.com', 443}}, {:inet, [:inet], {:tls_alert, {:unknown_ca, 'TLS client: In state wait_cert_cr at ssl_handshake.erl:2111 generated CLIENT ALERT: Fatal - Unknown CA\n'}}}]}}
    lib/tailwind.ex:348: Tailwind.fetch_body!/1
    lib/tailwind.ex:235: Tailwind.install/1
    (mix 1.14.3) lib/mix/task.ex:421: anonymous fn/3 in Mix.Task.run_task/4
    (mix 1.14.3) lib/mix/cli.ex:84: Mix.CLI.run_task/2
```

**Reply by SeanShubin** [reply]
I did check the marked answer but that does not apply to me because I am not using FreeBSD and I am not getting that error message, I am getting a message about an unknown Certificate Authority.

**Reply by drobban** [reply]
The reported issue is regarding the fact that there is no prebuilt binary for freebsd, so the question and answer do not apply for your situation.

@SeanShubin I would try and ask for help on #elixir@liberachat or on slack. The problem looks more like an "elixir" issue more then a tailwind one.
